tools/xlsx-mongo.js

```diff
@@ -23,28 +21,54 @@ program.parse(process.argv);
 
 // check path starts
 if (inputPath === undefined) {
-  console.error('Need the input xlsx spec file path!');
-  process.exit(1);
-}
-var suffix = inputPath.split('.').pop();
-if (suffix !== 'xlsx') {
-  console.error('File format must be xlsx.');
+  console.error('Need the input config file path!');
   process.exit(1);
 }
 var realPath = path.resolve(process.cwd(), inputPath);
 if (!fs.existsSync(realPath)) {
   console.log(realPath);
   console.error(realPath + ' does not exist.');
-  console.error('Please input a valid spec file path.');
+  console.error('Please input a valid config file path.');
   process.exit(1);
 }
 // check path end
 
 
+// check global.lconfig starts
+global.lconfig = require(realPath);
```

Comment:
avoid using global. you can use module to handle the dependency. A module to import all the configuration files, then require that module in other files.